.github/workflows/container_tests.yml

```diff
@@ -24,7 +23,7 @@ jobs:
         sudo apt-get update -yqq
         # for building Singularity images
         sudo apt-get install rpm
-        sudo apt-get install yum
+        sudo apt-get install dnf
```

Comment:
We should also replace `apt-get` with `apt` while we're at it...

Reply:
Is there a follow up action to do this?

Reply:
Fixed in 264b31f21